packages/init/src/index.ts

```diff
@@ -11,24 +11,32 @@ import {
   detectFramework,
   installLib,
 } from "./steps";
-import { findArgument, logs } from "./utils";
+import { findArgument, logs, getApplicationMode, Client } from "./utils";
 
 async function init() {
   const cwd = findArgument(process.argv, "cwd") || process.cwd();
-  const base = findArgument(process.argv, "base") || CONSTS.DEFAULT_BASE;
```

Comment:
better let QA know not to use the `base` flag

Reply:
Yes 😄  I planned to let them know once this PR is done and merged.
I'm sure this will make it easier for them 👍